home/src/main/java/com/devvailonge/home/presentation/FilmAdapter.kt

```diff
@@ -0,0 +1,56 @@
+package com.devvailonge.home.presentation
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import coil.load
+import com.devvailonge.home.R
+import com.devvailonge.home.presentation.FilmAdapter.FilmViewHolder
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+
+class FilmAdapter (private val films: List<HomeFragment.Film>)
```

Comment:
The whole app is about Tv Series, then I would rename this class for something like SeriesAdapter.